.github/workflows/build_image.yml

```
@@ -60,45 +70,61 @@
+        #         GIT_SSL_NO_VERIFY = "true"
+
+      - name: ${{ matrix.target.name }}
+        uses: docker/build-push-action@v6
         with:
           push: true
-          github-token: ${{ secrets.GITHUB_TOKEN }}
+          context: .
           file: ./docker/Dockerfile
-          build-args: ${{ matrix.target.build_args }}
           target: ${{ matrix.target.target_step }}
+          build-args: ${{ matrix.target.build_args }}
```

actionlint (1.7.4)

property "build_args" is not defined in object type {name: string; tag: string; target_step: string}

(expression)